lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -196,6 +197,71 @@ def test_create_ccx(self):
         self.assertEqual(response.status_code, 200)
         self.assertTrue(re.search('id="ccx-schedule"', response.content))
 
+    @SharedModuleStoreTestCase.modifies_courseware
+    @patch('ccx.views.render_to_response', intercept_renderer)
+    @patch('ccx.views.TODAY')
+    def test_get_ccx_schedule(self, today):
+        """
+        Get CCX schedule and checks number of blocks in it
+        """
+        def assert_elements_in_schedule(
```

Comment:
I dislike defining nested functions. There's no reason for this to not just be defined outside of this function, and then invoked within the test as `self.assert_elements_in_schedule`